osu.Game/OsuGame.cs

```diff
@@ -307,6 +307,10 @@ private void load()
             // Transfer any runtime changes back to configuration file.
             SkinManager.CurrentSkinInfo.ValueChanged += skin => configSkin.Value = skin.NewValue.ID.ToString();
 
+            BeatmapManager.PauseImports.BindTo(LocalUserPlaying);
+            SkinManager.PauseImports.BindTo(LocalUserPlaying);
+            ScoreManager.PauseImports.BindTo(LocalUserPlaying);
```

Comment:
This performs a two-way bind, which is slightly dangerous if we use `PauseImport`s elsewhere (as setting `PauseImport`s would then update `LocalUserPlaying`). Best to keep it updated via value change callback at least:
```suggestion
            LocalUserPlaying.ValueChanged += p =>
            {
                BeatmapManager.PauseImports.Value = p.NewValue;
                SkinManager.PauseImports.Value = p.NewValue;
                ScoreManager.PauseImports.Value = p.NewValue;
            };
```

Reply:
Yeah, seems safer. If we go with that solution I will change it back to being a non-bindable (as I originally implemented before deciding to switch), as there is no benefit from having it bindable.